ts/src/test/static/request/binance.json

```diff
@@ -2666,6 +2666,16 @@
                   }
                 ]
             }
+        ],
+        "fetchClosedOrdersWs": [
+            {
+                "description": "Fetch Closed Orders Ws",
+                "method": "fetchClosedOrdersWs",
+                "url": "https://binancefuture.com/dapi/v1/exchangeInfo",
```

Comment:
as you can see it makes no sense the test, because it's getting `exchangeInfo`. As of now we can't test ws methods